vue-frontend/src/components/contact/V2ContactLanding.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="background">
+    <div class="container tw-py-32">
+      <div class="v2-header-3-text canopas-gradient-text">Hey Smarty,</div>
+      <div class="v2-title-2-text tw-pt-20">
+        Thanks for reaching out to us. We're happy to help you achieve your
```

Comment:
Should reduce line height

Reply:
in this page OR change in app.vue file ??

Reply:
In this page